### PR TITLE
Move snippet to samples repo and add line highlighting

### PR DESCRIPTION
**Pull request by dksimpson**
## Summary

Related to .NET refresh work for task 1409933: https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1409933/

Move C# and VB code samples from article to samples repo and add highlighting: https://docs.microsoft.com/en-us/dotnet/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer
See comments in PR #https://github.com/dotnet/docs/pull/11110

### CODE REVIEW

File: snippets/csharp/VS_Snippets_VBCSharp/VbRadconService/CS/MyNewService.cs [hunk 1]
```diff
@@ -65,15 +65,15 @@ public MyNewService()
   //<Snippet3>
 	protected override void OnStart(string[] args)
 	{
-		eventLog1.WriteEntry("In OnStart");
+		eventLog1.WriteEntry("In OnStart.");
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_VBCSharp/VbRadconService/CS/MyNewService.cs]: This looks like it's formatted badly. Can we standardize on 4 spaces for indentation?

Comment (dksimpson) [reply snippets/csharp/VS_Snippets_VBCSharp/VbRadconService/CS/MyNewService.cs]: Hi @BillWagner, thanks for your review - I've standardized the spacing in the MyNewService C# and VB files.

**Comment by dksimpson** [comment]
#sign-off

**Comment by dksimpson** [comment]
This PR is related to #https://github.com/dotnet/docs/pull/11531, which can be merged after this PR is merged.
@rpetrusha

**Pull request merged**